cmake/SerialboxConfig.cmake

```diff
@@ -271,8 +196,7 @@ if(SERIALBOX_ROOT AND NOT(DEFINED SERIALBOX_NO_EXTERNAL_LIBS))
   #
   # NetCDF
   #
-  if(SERIALBOX_HAS_NETCDF)
-
+  if(SERIALBOX_NETCDF)
```

Comment:
Is there a good reason for this rename?

Reply:
the variables with the prefix `SERIALBOX_HAS_` refer to the config file generated by cmake. These variables are defined if the feature is available or undefined when the feature is not available. Since the cmake configuration file stores the variable `SERIALBOX_LOGGING`, `SERIALBOX_NETCDF` and `SERIALBOX_OPENSSL` with the content `ON` or `OFF` the rename made sense.
Should I rename them back to `SERIALBOX_HAS_`?